cartography/data/jobs/cleanup/gcp_iam_roles_cleanup.json

```diff
@@ -0,0 +1,29 @@
+{
+    "statements": [
+        {
+            "query": "MATCH (n:GCPIAMRole)<-[:RESOURCE]-(:GCPProject) WHERE n.lastupdated <> {UPDATE_TAG} WITH n LIMIT {LIMIT_SIZE} DETACH DELETE (n) return COUNT(*) as TotalCompleted",
```

Comment:
For all the cleanup queries, let's also filter GCPProject node to a specific project. See this for reference:
https://github.com/Cloudanix/cartography/pull/36/files#diff-e21ce887de4c4136633b58f57126608fe73076609d7a2ef83597d91fa6960e58R4